specs/WinRT/WinRTAPIContracts.md

```diff
@@ -98,16 +102,19 @@ The list of all contracts defined across Windows App SDK
-| DynamicDependency  | windowsappsdk | DynamicDependencyContract       | Microsoft.Windows.ApplicationModel.DynamicDependency |         |
-| EnvironmentManager | windowsappsdk | EnvironmentManagerContracct     | Microsoft.Windows.System                             |         |
-| MRTCore            | windowsappsdk | MrtContract                     | Microsoft.Windows.ApplicationModel.Resources         |         |
-| PowerNotifications | windowsappsdk | PowerNotificationsContract      | Microsoft.Windows.System.Power                       |         |
-| PushNotifications  | windowsappsdk | PushNotificationsContract       | Microsoft.Windows.PushNotifications                  |         |
-| WinUI              | winui         | WinUIContract                   | Microsoft.UI.Xaml                                    |         |
-| WinUI              | winui         | WinUIControlsContract           | Microsoft.UI.Xaml.Controls                           |         |
+| Feature            | Repository    | Contract                        | NamespaceMicrosoft.Windows.System.Power              | Comment  |
+|--------------------|---------------|---------------------------------|------------------------------------------------------|----------|
+| AccessControl      | windowsappsdk | AccessControlContract           | Microsoft.Windows.Security.AccessControl             |          |
+| AppLifecycle       | windowsappsdk | AppLifecycleContract            | Microsoft.Windows.AppLifecycle                       |          |
+| AppNotifications   | windowsappsdk | WindowsAppNotificationsContract | Microsoft.Windows.AppNotifications                   |          |
+| Deployment         | windowsappsdk | DeploymentManagerContract       | Microsoft.Windows.ApplicationModel.WindowsAppRuntime |          |
+| DynamicDependency  | windowsappsdk | DynamicDependencyContract       | Microsoft.Windows.ApplicationModel.DynamicDependency |          |
+| EnvironmentManager | windowsappsdk | EnvironmentManagerContracct     | Microsoft.Windows.System                             |          |
+| MRTCore            | windowsappsdk | MrtContract                     | Microsoft.Windows.ApplicationModel.Resources         |          |
+| PowerNotifications | windowsappsdk | PowerNotificationsContract      | Microsoft.Windows.System.Power                       |          |
+| PushNotifications  | windowsappsdk | PushNotificationsContract       | Microsoft.Windows.PushNotifications                  |          |
+| WinUI              | winui         | HostingContract                 | Microsoft.UI.Xaml.Hosting                            |          |
+| WinUI              | winui         | WinUIContract                   | Microsoft.UI.Xaml                                    |          |
+| WinUI              | winui         | WinUIControlsContract           | Microsoft.UI.Xaml.Controls                           |          |
```

Comment:
I don't actually see all of these contracts in metadata, at least not in 1.0. For example this one, and for example the XamlDirect feature below is in Windows but not WinAppSDK. All I see in metadata are
`MrtContract`
`WindowsAppSDKContract`
`WinUIContract`
`XamlContract`
?

Reply:
`MrtContract` was defined in `dev\MRTcore\mrt\Microsoft.Windows.AppliccationModel.Resources\src\Miccrosoft.Windows.ApplicationModel.Resources.idl` [2 years ago](https://github.com/microsoft/WindowsAppSDK/blame/31b83ed3e72512d37ebbab8d7d37f20f7cf9b44b/dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl)

`WindowsAppSDKContract` was defined recently which directly led to further discussion and then this spec. It may not be present in any build yet